Content.Server/Antag/AntagSelectionSystem.cs

```diff
@@ -298,6 +299,10 @@ public void MakeAntag(Entity<AntagSelectionComponent> ent, ICommonSession? sessi
 
         if (session != null)
         {
+            var antagLoadout = LoadoutSystem.GetJobPrototype(def.Loadout);
+            var pref = (HumanoidCharacterProfile) _pref.GetPreferences(session.UserId).SelectedCharacter;
```

Comment:
Does this even work because getjobprototype explicitly adds job in front of the role title and antagdefs should not have that.

Reply:
i made all the loadouts' names change to Loadout[RoleName] and changed the method